intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md

```diff
@@ -0,0 +1,47 @@
+# ssd300-fp32-onnx-0001
+
+## Use Case and High-Level Description
+
+This is an SSD model with a VGG16 backbone that is designed to perform object detection. The model has been pretrained on the VOC image database (VOC07+12 dataset).
+
+The model input is a blob that consists of a single image of "1x3x300x300" in RGB order.
+
+The model output for `ssd300-fp32-onnx-0001` is the usual object detection output for the 21 different classes present in the VOC database.
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection |
+| GFLOPs            | 62.815 |
+| MParams           | 26.285 |
+| Source framework  | PyTorch    |
+
+## Accuracy
+
+The quality metric (mAP) calculated on VOC07+12 validation dataset has the value of 78.28%.
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| mAP |         78.28% |
+
+## Input
+
+Image, shape - `1,3,300,300`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`
```

Comment:
model converted with `--reverse_input_channels` flag, so Channel order is `BGR`

Reply:
Done.